src/llm/eventToTags.ts

```diff
@@ -209,13 +249,23 @@ export async function addTagsToEvent(event: Event): Promise<string[]> {
     return results;
   }
 
+  // TODO: Update all types of tags to use SIPB LLMs endpoint in doCompletion
+  const contentTagPrompt = CONTENT_TAG_PROMPT.replace("{INSERT TITLE HERE}", event.title);
+
   const [formTags, contentTags, amenitiesTags] = await Promise.all([
     twoStagePrompt(FORM_TAG_PROMPT, EVENT_FORM_TAG_FUNCTION, ACCEPTABLE_FORM_TAGS),
-    twoStagePrompt(CONTENT_TAG_PROMPT, EVENT_CONTENT_TAG_FUNCTION, ACCEPTABLE_CONTENT_TAGS),
+    doCompletion(
+      `${contentTagPrompt}\n\`\`\`\n${text}\n\`\`\`\n\n---------------- Response --------------\n`,
+      EVENT_CONTENT_TAG_GRAMMAR
+    ),
```

Comment:
Would it be possible to implement the same kind of response that the GPT taggers provide? I think you'd have to add another field like "justification" to the grammar (or switch to a JSON schema since the SIPB LLM now supports JSON schemas) An example is below
```
----------Extracted Tags----------
{ form_tag: 'Study Break' }
----------Justification---------
Out of the the tags [Theater, Concert, Talk, Study Break, Movie Screening, Game, Sale, Rally, Dance, Party, Class Presentation], the most appropriate tag for the Gingerbread House Making event is "Study Break."

Here's the reasoning for each tag:

- **Theater**: This event does not involve any theatrical performance, so this tag does not apply.
- **Concert**: There is no musical performance or concert aspect to this event, so this tag does not apply.
- **Talk**: While there may be some informal discussions, the primary focus is on making gingerbread houses rather than a structured talk or workshop, so this tag does not apply.
- **Movie Screening**: There is no movie being screened at this event, so this tag does not apply.
- **Game**: Although there is a competition for the best gingerbread house, it is not a game in the traditional sense, so this tag does not apply.
- **Sale**: There is no fundraising or sale aspect mentioned in the event, so this tag does not apply.
- **Rally**: This event does not involve any form of protest or gathering for a cause, so this tag does not apply.
- **Dance**: There is no dancing involved in this event, so this tag does not apply.
- **Party**: While it has a social aspect, it is more focused on a specific activity (gingerbread house making) rather than a general party atmosphere, so this tag does not apply.
- **Class Presentation**: This event does not involve students presenting class projects, so this tag does not apply.
- **Study Break**: The email explicitly encourages students to take a break from studying and join the event, which aligns perfectly with the concept of a study break.

Therefore, the chosen tag is "Study Break" because the event is designed as a relaxing and enjoyable activity for students to take a break from their studies while engaging in a fun and creative task.
----------End Response----------
```